static/specification.html

```diff
@@ -236,7 +237,7 @@ <h2>
             with a non-zero amount, the provider notifies the browser:
               <ul>
                 <li>The <a>user agent</a> sets
-                <code>document.monetization.state</code> to
+                <code>navigator.monetization.state</code> to
```

Comment:
What states do we imagine this having and how will they be used in if we are collapsing the events, and in particular removing the stop event? 

Currently we have `pending | started | stopped`
We'll set the state to `stopped` but not have a corresponding event ?
We'll no longer have a `pending` event but set the `state` ?

Reply:
This reminds me of a conversation we had internally on slack with Matt Baer from write.as 
He wasn't happy just using a timeout to determine a user wasn't willing or able to monetize

Reply:
Hey, so we can add as many events as we need. The collapsing of events is just to merge the interface, but it doesn't need to limit the number of states we can have. 

>  We'll set the state to stopped but not have a corresponding event ?

No, let me clarify with an example 

```
navigator.monetization.state; // changes to "stopped"
navigator.monetization.onstatechage = event => {
     switch(navigator.monetization.state):
            case "stopped":
                 // do stuff..
            case "started":
                 // do stuff..
            case "whatever-we-need":
                // do stuff..
            default: 
                // do default stuff..
}
```

Reply:
Got it! 
In the latest extension, I mapped pending/started to the proposed 'interactive', and stopped to 'idle'
Would be good if we can agree on some terms @ https://github.com/WICG/webmonetization/issues/220

I'm pretty sure @sharafian would like to see as little change as possible